modules/serverless-ossm-installing-and-configuring-openshift-service-mesh.adoc

```diff
@@ -0,0 +1,148 @@
+:_content-type: PROCEDURE
+[id="serverless-ossm-installing-and-configuring-openshift-service-mesh_{context}"]
+= Installing and configuring {SMProductShortName}
+
+To integrate {ServerlessProductShortName} with {SMProductShortName}, you need to install {SMProductShortName} with a specific configuration.
+
+.Procedure
+
+. Create a `ServiceMeshControlPlane` resource in the `istio-system` namespace with the following configuration:
++
+[IMPORTANT]
+====
+If you have an existing `ServiceMeshControlPlane` object, make sure that you have the same configuration applied.
+====
++
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  profiles:
+  - default
+  security:
+    dataPlane:
+      mtls: true <1>
+  techPreview:
+    meshConfig:
+      defaultConfig:
+        terminationDrainDuration: 35s <2>
+  gateways:
+    ingress:
+      service:
+        metadata:
+          labels:
+            knative: ingressgateway <3>
+  proxy:
+    networking:
+      trafficControl:
+        inbound:
+          excludedPorts: <4>
+          - 8444 # metrics
+          - 8022 # serving: wait-for-drain k8s pre-stop hook
+----
+<1> Enforce strict mTLS in the mesh. Only calls using a valid client certificate are allowed.
+<2> {ServerlessProductShortName} has a graceful termination for Knative Services of 30 seconds. `istio-proxy` needs to have a longer termination duration to make sure no requests are dropped.
+<3> Define a specific selector for the ingress gateway to target only the Knative gateway.
+<4> These ports are called by Kubernetes and cluster monitoring, which are not part of the mesh and cannot be called using mTLS. Therefore, these ports are excluded from the mesh.
+
+. Add the namespaces that you would like to integrate with {SMProductShortName} to the `ServiceMeshMemberRoll` object as members:
```

Comment:
For a follow-up: I noticed earlier there were a lot of "part of the `ServiceMeshMemberRoll`" or "member of the `ServiceMeshMemberRoll`" without an "object" on the end, but I didn't call out as I wasn't sure of the implication. But it looks like here that you've got "object" on the end, so suggest going back to the others and adding for consistency as well.